vignettes/tidytranscriptomics.Rmd

```diff
@@ -492,50 +501,31 @@ With stripcharts we can see if replicates tend to group together and how the exp
 
 
 ```{r}
-counts_scaled %>%
+strip_chart = 
+	counts_scaled %>%
 	
 	# extract counts for top DE genes
 	filter(transcript %in% topgenes_symbols) %>%
 	
 	# make stripchart
-	ggplot(aes(x = dex, y = counts_scaled + 1, fill = dex)) +
-	geom_boxplot() +
-	geom_jitter() +
-	facet_wrap(~transcript) +
-	scale_y_log10()+
-	theme_bw()
-```
-
-### Interactive Plots
-
-A really nice feature of using tidyverse and ggplot2 is that we can make interactive plots quite easily using the plotly package. This can be very useful for exploring what genes or samples are in the plots. To make interactive plots we first save our ggplot2 object, for example as "p". We can add label= into the `aes` if we want other variables, such as the sample name to be visible when we hover over the plot.
-
-```{r}
-p <- counts_scaled %>%
-	
-	# extract counts for top DE genes
-	filter(transcript %in% topgenes_symbols) %>%
-	
-  # make stripchart
 	ggplot(aes(x = dex, y = counts_scaled + 1, fill = dex, label = sample)) +
 	geom_boxplot() +
 	geom_jitter() +
 	facet_wrap(~transcript) +
 	scale_y_log10()+
 	theme_bw()
+
+strip_chart
 ```
 
-Then we simply give our plot, p, to the ggplotly function. We can also specify which parameters from the aes() we want to show up when we hover over the plot with `tooltip`.
+### Interactive Plots
 
-```{r}
-ggplotly(p, tooltip = c("label", "y"))
-```
+A really nice feature of using tidyverse and ggplot2 is that we can make interactive plots quite easily using the plotly package. This can be very useful for exploring what genes or samples are in the plots. We can make interactive plots directly from our ggplot2 object (strip_cart). Having label= into the `aes` is useful to visualise the identified of the data point or other variables to be visible when we hover over the plot.
```

Comment:
```suggestion
A really nice feature of using tidyverse and ggplot2 is that we can make interactive plots quite easily using the plotly package. This can be very useful for exploring what genes or samples are in the plots. We can make interactive plots directly from our ggplot2 object (strip_chart). Having label= into the `aes` is useful to visualise the identified of the data point or other variables to be visible when we hover over the plot.
```